linux.js

```diff
@@ -43,6 +45,11 @@ module.exports = {
     function renameElectronBinary () {
       fs.rename(originalBinary, finalBinary, function electronRenamed (err) {
         if (err) return cb(err)
+        if (opts.asar) {
+          asarApp(cb)
+        } else {
+          cb()
```

Comment:
I thought so as it didn't seem to call `cb()` which for the cli probably doesn't matter but if someone was using it as a lib they would want to know when the `cb()` was called.